src/tactic/expand_exists.lean

```diff
@@ -0,0 +1,169 @@
+/-
+Copyright (c) 2022 Ian Wood. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Ian Wood
+-/
+import init.meta.declaration
+import meta.expr
+import init.meta.lean.parser
+
+/-!
+# `expand_exists`
+
+`expand_exists` is an attribute which takes a proof that something exists with some property, and
+outputs a value using `classical.some`, and a proof that it has that property using
+`classical.some_spec`.
```

Comment:
Adding a small example here would be nice, perhaps just the one from the original issue